src/Microsoft.Fx.Portability/ObjectModel/AnalyzeRequest.cs

```diff
@@ -37,7 +37,8 @@ public sealed class AnalyzeRequest : IComparable
         public int CompareTo(object obj)
         {
             var analyzeObject = obj as AnalyzeRequest;
-            return ApplicationName.CompareTo(analyzeObject.ApplicationName);
+
+            return string.Compare(ApplicationName, analyzeObject.ApplicationName, StringComparison.CurrentCulture);
```

Comment:
StringComparison.CurrentCulture (as anything that is culture specific) should be used when the strings are being displayed to the user. If it's internal to the program, ordinal should be used.